community/zsh/sources

```diff
@@ -0,0 +1,3 @@
+https://sourceforge.net/projects/zsh/files/zsh/5.7.1/zsh-5.7.1.tar.xz
+patches/remove_bin_duplicate.patch
+files/zprofile
```

Comment:
Is shipping a `zprofile` file a typical part of packaging `zsh`?

Reply:
It seems so. I know Gentoo, Arch and even Alpine provide a zprofile. The one I included will just include `/etc/profile`.